src/git.sh

```
@@ -2,41 +1,60 @@
+  local gitea=$(echo "$args_json" | jq -r '.gitea // "false"')
+
+  git config user.name "GitHub Actions"
+  git config user.email "actions@github.com"
+  if [ ! -z "$GITHUB_WORKSPACE" ]; then
+    git config --global --add safe.directory "$GITHUB_WORKSPACE"
+  fi
+}
 
 git_get_commit_message() {
+  # Parse arguments using the new parse_arguments function
+  local args_json=$(parse_arguments "$@")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)